product.go

```diff
@@ -23,8 +23,28 @@ type ProductServiceOp struct {
 
 // Product represents a Shopify product
 type Product struct {
-	ID    int    `json:"id"`
-	Title string `json:"title"`
+	ID             int             `json:"id"`
+	Title          string          `json:"title"`
+	BoldHTML       string          `json:"body_html"`
+	Vendor         string          `json:"vendor"`
+	ProductType    string          `json:"product_type"`
+	Handle         string          `json:"handle"`
+	CreatedAt      string          `json:"created_at"`
+	UpdatedAt      string          `json:"updated_at"`
+	PublishedAt    string          `json:"published_at"`
```

Comment:
Could you add these three dates as `*time.Time` instead, similar to the [order dates](https://github.com/getconversio/go-shopify/blob/64825aa463f6dd5c7e81a0e15a402f807178e117/order.go#L50-L53)?